openpype/hosts/unreal/api/pipeline.py

```diff
@@ -133,6 +152,31 @@ def ls():
         yield data
 
 
+def lsinst():
```

Comment:
I would probably put underscore there
```suggestion
def ls_inst():
```